src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocVariableCheck.java

```diff
@@ -176,6 +167,17 @@ private boolean isIgnored(DetailAST ast) {
             || "serialVersionUID".equals(name);
     }
 
+    /**
+     * Checks whether a method has the correct access modifier to be checked.
+     *
+     * @param accessModifier the access modifier of the method.
+     * @return whether the method matches the expected access modifier.
+     */
+    private boolean matchAccessModifiers(final AccessModifierOption accessModifier) {
```

Comment:
```suggestion
    private boolean matchAccessModifiers(AccessModifierOption accessModifier) {
```

Please remove `final` from the parameter declaration, this is an outdated practice.

Reply:
removed